source/modulo_components/include/modulo_components/ComponentInterface.hpp

```diff
@@ -684,46 +696,81 @@ template<class NodeT>
 inline void ComponentInterface<NodeT>::add_tf_listener() {
   if (this->tf_buffer_ == nullptr || this->tf_listener_ == nullptr) {
     RCLCPP_DEBUG(this->get_logger(), "Adding TF buffer and listener.");
+    console_bridge::setLogLevel(console_bridge::CONSOLE_BRIDGE_LOG_NONE);
     this->tf_buffer_ = std::make_shared<tf2_ros::Buffer>(this->get_clock());
     this->tf_listener_ = std::make_shared<tf2_ros::TransformListener>(*this->tf_buffer_);
   } else {
     RCLCPP_DEBUG(this->get_logger(), "TF buffer and listener already exist.");
   }
 }
 
+template<class NodeT>
+inline void ComponentInterface<NodeT>::activate_tf_broadcaster() {
+  if (this->publisher_type_ == modulo_new_core::communication::PublisherType::PUBLISHER) {
```

Comment:
Small nitpick but I think this may be safer / more readable
```suggestion
  if (this->publisher_type_ != modulo_new_core::communication::PublisherType::LIFECYCLE_PUBLISHER) {
```